src/wp-includes/blocks/index.php

```diff
@@ -12,6 +12,63 @@
 require BLOCKS_PATH . 'widget-group.php';
 require BLOCKS_PATH . 'require-dynamic-blocks.php';
 
+/**
+ * Registers core block style handles.
+ *
+ * While {@see register_block_style_handle()} is typically used for that, the way it is
+ * implemented is inefficient for core block styles. Registering those style handles here
+ * avoids unnecessary logic and filesystem lookups in the other function.
+ *
+ * @since 6.3.0
+ */
+function register_core_block_style_handles() {
+	if ( ! wp_should_load_separate_core_block_assets() ) {
+		return;
+	}
+
+	static $core_blocks_meta;
+	if ( ! $core_blocks_meta ) {
+		$core_blocks_meta = require ABSPATH . WPINC . '/blocks/blocks-json.php';
+	}
+
+	$includes_url = includes_url();
```

Comment:
```suggestion
	$includes_url = includes_url( '', 'relative' );
```
Not blocker. before this changes it was `/wp-includes/blocks/blockbame/style.css, if we use `includes_url()` it return include URL link.

Reply:
This was added becuase of feedback from @felixarntz https://github.com/WordPress/wordpress-develop/pull/4621#discussion_r1232574807

I don't see how it hurts to include the url here.